#HoCVectorArray.msg

Header header
HoCVector[] vectors
